crest/Assets/Crest/Crest/Scripts/Shapes/OceanWaveSpectrum.cs

```diff
@@ -104,7 +107,7 @@ public float GetAmplitude(float wavelength, float componentsPerOctave)
             var alpha = (wavelength - lower) / lower;
 
             // Power
-            var pow = Mathf.Lerp(_powerLog[index], _powerLog[Mathf.Min(index + 1, _powerLog.Length - 1)], alpha);
+            var pow = hasNextIndex ? Mathf.Lerp(thisPower, nextPower, alpha) : _powerLog[index];
```

Comment:
After pushing my last change, I've just noticed the hastNextIndex check here. Is this required? I was thinking that the two powers should always be lerped, but one or both of them are pushed down to min if disabled. So disabling a slider forces it to min power, but nothing else changes. To get to the point, can we remove the ternary on hasNextIndex from this line?

Reply:
It was to not interpolate anything for the last slider (512) since there is no next power. So you want interpolation with `MIN_POWER_LOG` for that case too? Before it was interpolating power 512 with power 512.

Reply:
You're right, it was confusion on my part. I mistakenly thought hasNextIndex included the check for the disabled flag, but its purely based on index, so i think we can disregard this